test/end-to-end/core.sh

```diff
@@ -129,7 +129,7 @@ os::cmd::expect_success 'oc env -n default dc/docker-registry REGISTRY_MIDDLEWAR
 os::cmd::expect_success 'oc rollout status dc/docker-registry'
 os::log::info "Restore configured to enable mirroring"
 
-registry_pod="$(oc get pod -n default -l deploymentconfig=docker-registry --show-all=false --template='{{(index .items 0).metadata.name}}')"
+registry_pod="$(oc get pod -n default -l deploymentconfig=docker-registry --template '{{range .items}}{{if not .metadata.deletionTimestamp}}{{.metadata.name}}{{end}}{{end}}')"
```

Comment:
The rollout does not synchronize with the deletion. I think excluding pods which are in deletion state is correct.

We shouldn't have more than one active pod if our instance=1 deployment works. If not, the test breaks. We could add a space and select the first pod, but that would hide a problem.